### DISCUSSION TITLE
Correct parameters for MQTT login

### DISCUSSION THREAD

**Discussion by mauimauer** [mikey0000/PyMammotion#18] [Q&A] [answered]
It seems the app is no longer plainly outputting all the parameters required for MQTT connection.

I have access to the shared prefs and SQLite DBs of the app. Can anyone point me which parameters to use.

The following seem to be promising:
productId
productKey (starts with a1)
productSecret
deviceName (has a Luba- prefix)
iotId (starts with AMQ)
identityId

**Comment by mikey0000** [comment, marked as answer]
Product key and secret

26/05/2024 11:11:11 pm Sebastian Mauer ***@***.***>:
